package/contents/ui/main.qml

```diff
@@ -183,13 +196,40 @@ PlasmaCore.Dialog {
                     return false;
                 }
 
-                WindowLayout {
-                    Loader {
-                        id: layoutFile
-                        source: fileUrl
+                ColumnLayout {
+                    id: column
+                    function childHasFocus() {
+                        return layout.childHasFocus();
+                    }
+
+                    Text {
+                        id: label
+                        text: layoutFile.item.name
+                        color: "white"
+                        font.pointSize: 15
```

Comment:
This should be a `PlasmaComponents.Label` instead of `Text`. Also remove the color and font size properties.

Reply:
A preview with some edits:

![image](https://github.com/qewer33/Exquisite/assets/69015181/6dbb1347-e460-4c95-9fa3-38b752a07c48)

```qml
// Kirigami import at the top of the file
import org.kde.kirigami 2.20 as Kirigami

// Text changed to a Label and moved beloew the layout
PlasmaComponents.Label {
    id: label
    text: layoutFile.item.name
    color: Kirigami.Theme.disabledTextColor
    width: layout.width
    elide: Text.ElideRight
    Layout.alignment: Qt.AlignHCenter
}
```

Reply:
This is f***ing beautiful. I love it.

Reply:
Oh, I really like it after those changes.

I had to do the color because with just Text, it came out dark on dark in night mode, which was unreadable.

Reply:
Oh, I really like it after those changes.

I had to do the color because with just Text, it came out dark on dark in night mode, which was unreadable.

Anyhow, change made, though I made the location of the name a configuration setting. :)